plugins/modules/proxmox_snap_info.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, Jeffrey van Pelt (@Thulium-Drake) <jeff@vanpelt.one>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: proxmox_snap_info
+short_description: Snapshot management of instances in Proxmox VE cluster
+version_added: 6.3.0
+description:
+  - Lists all snapshots from instances in Proxmox VE cluster.
+  - to be optionally combined with older_than
+  - to be optionally combine with snapname (finds snapshot starting with the snapname)
```

Comment:
Paragraphs should always consist of full sentences. These two paragraphs have sentences that start with a lower-case letter and do not end with a period or any other end of sentence mark. (Also I have no idea what `older_than` is. If you mean the option of this module, use `I(...)` - see https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation.)